integ_test/contract.js

```diff
@@ -31,46 +34,76 @@ function deployCt(ct, ctData, args, cb) {
         //.on('confirmation', (confirmationNumber, receipt) => { console.log(confirmationNumber, receipt) });
 }
 
-
 describe('contracts', () => {
   let opts = { 
       from: test_cfg.TEST_ACCT_ADDR,
       gas: test_cfg.GAS,
       gasPrice: test_cfg.GAS_PRICE,
   };
 
-  let typesBin
-  let typesAbi
   let ct;
   let ctInstAddress;
 
+  let web3 = new Web3(client);
+
+  let typesSrc;
+  let compiled;
+  let BIN;
+  let ABI;
+  let contract;
+
   let testNumber = 31337;
+  let source = fs.readFileSync(typesSrcPath, 'utf-8');
```

Comment:
not mandatory, but i think it'd be cleaner if we put this inside the "steps" Object and use async.apply over fs.readFile instead of fs.readFileSync

Reply:
updated now!